psi4/driver/driver_findif.py

```diff
@@ -443,9 +503,8 @@ def _geom_generator(mol: Union["qcdb.Molecule", core.Molecule], freq_irrep_only:
         Number of points to evaluate at for each displacement basis vector. Count
         includes the central reference point.
 
-    displacement_space : {'CdSalc'}
```

Comment:
I think leave `displacement_space` alone and add `engine` separately (with `internal | molsym`). `CdSalc` was so this could be expanded to `Cartesian` or something someday.